nzedb/ConsoleTools.php

```diff
@@ -135,12 +135,14 @@ public function convertTime($seconds)
 	{
 
 		if ($seconds > 3600) {
-			return round($seconds / 3600) . " hour(s)";
-		} else if ($seconds > 60) {
-			return round($seconds / 60) . " minute(s)";
-		} else {
-			return $seconds . " second(s)";
+			return round($seconds / 3600) . ' hour(s)';
 		}
+
+		if ($seconds > 60) {
+			return round($seconds / 60) . ' minute(s)';
+		}
+
+		return $seconds . ' second(s)';
```

Comment:
This still leaves us with three return points. To reduce code complexity we should try to go for a single return point.